### PR TITLE
feat: get looperlands up and running

### PR DESCRIPTION
**Pull request by yashu2001**

### CODE REVIEW

**Review by balkshamster — COMMENTED** [review]
Why did you change the gnashling?

File: Dockerfile [hunk 1]
```diff
@@ -6,13 +6,14 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt -y update
 RUN apt -q -y install curl
 RUN apt-get update -yq \
-    && apt-get install curl gnupg python2 -yq \
+    && apt-get install curl gnupg python2 libxml2-dev libxslt1-dev zlib1g-dev gcc-aarch64-linux-gnu g++-aarch64-linux-gnu python2.7-dev -yq \
```
Comment (balkshamster) [inline Dockerfile:9]: Why did you install these?

**Comment by groot406** [comment]
> Why did you change the gnashling?

I think the casing is wrong Gnashling vs gnashling

**Comment by ordinaryadam** [comment]
> > Why did you change the gnashling?
> 
> I think the casing is wrong Gnashling vs gnashling

Ah this was a duplicate file that's already been removed in a commit, both were present but the upper casing was removed already - thanks for updating though!

**Pull request closed without merging**